myvue4/src/views/Signup.vue

```diff
@@ -0,0 +1,91 @@
+
+<template>
+  <div class="form">
+    
+    <img alt="Vue logo" src="../assets/logo.png">
+
+    <h1>新規登録画面</h1>
+    <table>
+      <tr>
+        <td>ユーザー名</td>
+        <td><input type="text" placeholder="userName" v-model='name'></td>
+      </tr>
+      <tr>
+        <td>メールアドレス</td>
+        <td><input type="text" placeholder="E-mail" v-model='email'></td>
+      </tr>
+      <tr>
+        <td>パスワード</td>
+        <td><input type="text" placeholder="Password" v-model='password'></td>
+      </tr>  
+    </table>
+
+    <button @click="signUp" class="link-signup">新規登録</button>
+    <router-link class="link-signin" to="/signin">ログインはこちらから</router-link>
+
+  
+
+  </div>
+</template>
+
+
+<script>
+
+export default{
+  name:'Signup',
+  data(){
+    return{
+      name:'',
```

Comment:
細かい点になりお伝えするのが少し遅かったかもしれませんが、`:`の後にはスペースが必要です。
```suggestion
      name: '',
```

エディタの自動修正を使用するとこのようなミスがなくなりますので、参考にしてみてください。
現場ですとほぼ必ず使用されています。
https://qiita.com/mitashun/items/e2f118a9ca7b96b97840